### PR TITLE
feat(trs-filer): create TRS-Filer component

### PR DESCRIPTION
**Pull request by ltwmori**
## Description

This PR implements the TRS-filer component as described in the linked issue.
Fixes issue #97 

## Checklist

- [ ] My code follows the [contributing guidelines][contributing] of this project.
- [ ] I am aware that all my commits will be squashed into a single commit, using the PR title as the commit message.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have updated the user-facing documentation to describe any new or changed behavior.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have not reduced the existing code coverage.

## Comments

[contributing]: CONTRIBUTING.md

### CODE REVIEW

File: packages/ecc-client-elixir-trs-filer/.eslintrc.cjs
Comment (SalihuDickson) [inline packages/ecc-client-elixir-trs-filer/.eslintrc.cjs:1]: how come you have 2 eslint files?

File: packages/ecc-client-elixir-trs-filer/README.md
Comment (SalihuDickson) [inline packages/ecc-client-elixir-trs-filer/README.md:1]: this was probably modified by the FAST cli, please revert this and then add your component to the component table

**Review by SalihuDickson — CHANGES_REQUESTED** [review]

**Comment by SalihuDickson** [comment]
since this is built on top of the trs component, you should have created this branch from the trs branch so that changes you make there can be pulled into this branch.  You probably don't want to hear this, but I think you should restart this and do that instead 😬

**Comment by ltwmori** [comment]
> since this is built on top of the trs component, you should have created this branch from the trs branch so that changes you make there can be pulled into this branch. You probably don't want to hear this, but I think you should restart this and do that instead 😬

@SalihuDickson ok! it's not difficult for me to address this. I'll get to it right after I complete resolving the comments on the [TRS PR](https://github.com/elixir-cloud-aai/cloud-components/pull/55). Thanks for your review!

File: packages/ecc-client-elixir-trs-filer/.eslintrc.cjs
Comment (ltwmori) [reply packages/ecc-client-elixir-trs-filer/.eslintrc.cjs:1]: This is the problem of [FAST cli](https://microsoft.github.io/fast-cli/docs/initialize/). I used `npm init @microsoft/fast-project` and it automatically created these 2 files (the same thing happened in the TRS code). Sure thing, I'll remove one eslint file. No need to worry.

**Pull request closed without merging**